### ISSUE TITLE
Documentation of maven plugins

### ISSUE THREAD

**Issue by laeubi** [apache/felix-atomos#65]
According to the examples Atomis provides some maven plugins but I can't find a documentation about what they do, used parameters and so on. Is there a site-documentation deployed somewhere? Some readme I'm missing?

**Comment by tjwatson** [comment]
This is true.  The maven plugins are not really ready to be considered for a 1.0.0 release.  Much work is needed to polish the implementation and document the plugin.  Others need to pick up the work here to finish what @stbischof had started with the maven plugins.
